### PR TITLE
Vibhuti

### PR DESCRIPTION
**Pull request by VibhutiBansal-11**
### Figure Number

Book 1 - 10.6
### Figures

[Gist](https://gist.github.com/VibhutiBansal-11/5f8cbdbadee11beb1290af3359df2598)
### Issue 

[Figure 10.6](https://github.com/probml/pml-book/issues/323) 
## Steps

 - [x] Changed the name of the variables/functions to PEP 8 style
  - [x] Added code to generate two figures as given in the book and Latexified them.
  - [x] Converted code to JAX
## Checklist:

- [x] Performed a self-review of the code
- [x] Tested on Google Colab.

### CODE REVIEW

File: notebooks/book1/10/logreg_poly_demo.ipynb [hunk 1]
```diff
@@ -0,0 +1,339 @@
+{
```
Comment (karm-patel) [inline notebooks/book1/10/logreg_poly_demo.ipynb:1]: <p>Title please</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (karm-patel) [inline notebooks/book1/10/logreg_poly_demo.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    latexify(width_scale_factor=2, fig_height=2)
</pre><p>Set height = 1.85</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (karm-patel) [inline notebooks/book1/10/logreg_poly_demo.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    train_plot()
</pre><p>Can we get rid out of this warnings?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

**Comment by karm-patel** [comment]
@VibhutiBansal-11, Plot looks good to me but are you sure the above "After PR" figure is latexified,  I doubted because the font size looks small.

I have added a few nits, after solving them you can go ahead.

**Comment by VibhutiBansal-11** [comment]
@karm-patel  I have updated the required changes according to comments. Kindly have a look . This is the updated  [Gist](https://gist.github.com/VibhutiBansal-11/5f8cbdbadee11beb1290af3359df2598)

**Pull request merged**